modules/standard/Regexp.chpl

```diff
@@ -18,12 +18,302 @@
+
+  ASCII character classes::
+
+  [:alnum:]    alphanumeric (== [0-9A-Za-z])
+  [:alpha:]    alphabetic (== [A-Za-z])
+  [:ascii:]    ASCII (== [\x00-\x7F])
+  [:blank:]    blank (== [\t ])
+  [:cntrl:]    control (== [\x00-\x1F\x7F])
+  [:digit:]    digits (== [0-9])
+  [:graph:]    graphical (== [!-~] == 
+                 [A-Za-z0-9!"#$%&'()*+,\-./:;<=>?@[\\\]^_`{|}~])
+  [:lower:]    lower case (== [a-z])
+  [:print:]    printable (== [ -~] == [ [:graph:]])
+  [:punct:]    punctuation (== [!-/:-@[-`{-~])
+  [:space:]    whitespace (== [\t\n\v\f\r ])
+  [:upper:]    upper case (== [A-Z])
+  [:word:]     word characters (== [0-9A-Za-z_])
+  [:xdigit:]   hex digit (== [0-9A-Fa-f])
+
+  Unicode character class names--general category:
+  C            other
+  Cc           control
+  Cf           format
+  Co           private use
+  Cs           surrogate
+  L            letter
+  Ll           lowercase letter
+  Lm           modifier letter
+  Lo           other letter
+  Lt           titlecase letter
+  Lu           uppercase letter
+  M            mark
+  Mc           spacing mark
+  Me           enclosing mark
+  Mn           non-spacing mark
+  N            number
+  Nd           decimal number
+  Nl           letter number
+  No           other number
+  P            punctuation
+  Pc           connector punctuation
+  Pd           dash punctuation
+  Pe           close punctuation
+  Pf           final punctuation
+  Pi           initial punctuation
+  Po           other punctuation
+  Ps           open punctuation
+  S            symbol
+  Sc           currency symbol
+  Sk           modifier symbol
+  Sm           math symbol
+  So           other symbol
+  Z            separator
+  Zl           line separator
+  Zp           paragraph separator
+  Zs           space separator
+
+  Unicode character class names--scripts (with explanation where non-trivial):
+  Arabic
+  Armenian
+  Balinese
+  Bengali
+  Bopomofo
+  Braille
+  Buginese
+  Buhid
+  Canadian_Aboriginal
+  Carian
+  Cham
+  Cherokee
+  Common       characters not specific to one script
+  Coptic
+  Cuneiform
+  Cypriot
+  Cyrillic
+  Deseret
+  Devanagari
+  Ethiopic
+  Georgian
+  Glagolitic
+  Gothic
+  Greek
+  Gujarati
+  Gurmukhi
+  Han
+  Hangul
+  Hanunoo
+  Hebrew
+  Hiragana
+  Inherited    inherit script from previous character
+  Kannada
+  Katakana
+  Kayah_Li
+  Kharoshthi
+  Khmer
+  Lao
+  Latin
+  Lepcha
+  Limbu
+  Linear_B
+  Lycian
+  Lydian
+  Malayalam
+  Mongolian
+  Myanmar
+  New_Tai_Lue  aka Simplified Tai Lue
+  Nko
+  Ogham
+  Ol_Chiki
+  Old_Italic
+  Old_Persian
+  Oriya
+  Osmanya
+  Phags_Pa
+  Phoenician
+  Rejang
+  Runic
+  Saurashtra
+  Shavian
+  Sinhala
+  Sundanese
+  Syloti_Nagri
+  Syriac
+  Tagalog
+  Tagbanwa
+  Tai_Le
+  Tamil
+  Telugu
+  Thaana
+  Thai
+  Tibetan
+  Tifinagh
+  Ugaritic
+  Vai
+  Yi
+
+  Vim character classes:
+  \d      digits (== [0-9])
+  \D      not «\d»
+  \w      word character
+  \W      not «\w»
```

Comment:
Mentioning the code section header one more time, not because I think you need to hear the justification but as a helpful pointer to all the places I think it is useful.